tests/test_cli.py

```diff
@@ -807,6 +807,26 @@ def test_version(capfd):
         assert out == __version__
 
 
+@pytest.mark.parametrize(
+    "study,msg",
+    [("core", "distributed_with"), ("study_valid", "version"), ("not_a_study", "not an installed")],
```

Comment:
Maybe use `version 1.0.0` to confirm that we actually do pull the version number from the study